src/PopperWrapper/index.jsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2019 dunnhumby Germany GmbH.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the LICENSE file
+ * in the root directory of this source tree.
+ *
+ */
+
+/* global document */
+
+import React                                      from 'react';
+import ReactDOM                                   from 'react-dom';
+import { Manager, Reference, Popper }             from 'react-popper';
+import PropTypes                                  from 'prop-types';
+
+const PopperWrapper = props =>
+{
+    const {
+        children,
+        container,
+        popper,
+        popperOffset,
+        popperPosition,
+    } = props;
+
+    let offset = '';
+
+    switch ( popperOffset )
+    {
+    case 'S':
+        offset = '8px';
+        break;
+    case 'M':
+        offset = '16px';
+        break;
+    case 'L':
+        offset = '24px';
+        break;
+    case 'XL':
+        offset = '32px';
+        break;
+    default:
+        offset = '16px';
+    }
+
+    return (
+        <Manager>
+            <Reference>
+                { ( { ref } ) => (
+                    <div ref = { ref }>
+                        { children }
+                    </div>
+                ) }
+            </Reference>
+            { ReactDOM.createPortal(
+                <Popper
+                    placement = { popperPosition }
+                    modifiers = { { offset: { offset: `0, ${offset}` } } }>
+                    { ( { ref, style } ) => (
+                        <div
+                            ref   = { ref }
+                            style = { style }>
+                            { popper }
+                        </div>
+                    ) }
+                </Popper>,
+                container ? document.querySelector( container ) :
+                    document.body,
+            ) }
+        </Manager>
+    );
+};
+
+PopperWrapper.propTypes =
+{
+    /**
+     *  Dom element to attach popper
```

Comment:
I think this should be something like _Reference node_  (the prop accepts a React node, not a DOM element)